vendor/github.com/moby/vpnkit/go/pkg/vmnet/vmnet.go

```diff
@@ -563,7 +563,9 @@ func (v *Vif) dhcp() (net.IP, error) {
 	ethernet := NewEthernetFrame(broadcastMAC, v.ClientMAC, 0x800)
 	ethernet.setData(ipv4.Bytes())
 
-	file, err := os.Create("/tmp/go.pcap")
+	runDir := os.Getenv("XDG_RUNTIME_DIR")
+	pcapPath := fmt.Sprintf("%s/go.pcap", runDir)
```

Comment:
ioutil.TempFile?

Reply:
From [https://golang.org/src/io/ioutil/tempfile.go](https://golang.org/src/io/ioutil/tempfile.go):
> It is the caller's responsibility to remove the file when no longer needed.

Perhaps it could be deleted in the Close method, but I don't know the exact purpose of the .pcap file.  Both in the original and in the PR, the same .pcap file is opened by the user UID outside the namespace and by UID 0 within the namespace - is the fact that both share the pcap file critical to the function of rootlesskit (I hope that it's obvious that I don't know what this file does).